runtime/src/main/java/com/flipkart/flux/guice/module/ContainerModule.java

```diff
@@ -65,7 +67,7 @@
 	 */
 	@Override
 	protected void configure() {
-
+		bind(MetricsClient.class).to(MetricsClientImpl.class);
```

Comment:
Make it singleton
bind(MetricsClient.class).to(MetricsClientImpl.class).in(Singleton.class);